### DISCUSSION TITLE
Add watch paths to docker compose build packs?

### DISCUSSION THREAD

**Discussion by DanielBiegler** [coollabsio/coolify#4331] [Feature Requests]
Heya, love the project -- thank you!

I was wondering if there is a specific reason (other than low prio todo) why a git based deployment with a docker-compose file has no configurable watch path. For now (beta 367) Coolify starts redeploying my composed stacks even though the files dont change. Is this really necessary?

Tried looking for similar issues/discussions and just found a comment asking for this too [here](https://github.com/coollabsio/coolify/discussions/3512#discussioncomment-10965724).

**Comment by pesickadavid** [comment]
Hi, this would be really usefull!

I have a "infra" git repo that has many `docker-compose.yaml` files that should be redeployed only if I change their contents.
For basic Dockerfile git deployments this works like a charm! But is missing for Docker Compose deployments

**Comment by DanielBiegler** [comment]
This is especially useful for monorepos. For example you have multiple compose files for different services in a single repo that currently _(v4-beta.422)_ get rebuilt and redeployed unnecessarily every time. 

Edit: Ahh, just saw that David mentioned this already oops. Anyway, there's still interest from my side for this feature, it just came up in a discussion I had with someone.

**Comment by DanielBiegler** [comment]
Hey @pesickadavid this finally got added in the latest release via #6598 !

Just tried it out and its glorious! :heart: 

Thank you @ossamalafhel and @andrasbacsai I appreciate you!

**Reply by pesickadavid** [reply]
Ooh nice! Thanks for notifying me!

Also thanks @ossamalafhel  and @andrasbacsai  for the implementation!
Gonna test it tommorow 🙌

**Reply by ossamalafhel** [reply]
@DanielBiegler @pesickadavid  my pleasure and thx to @andrasbacsai
